src/navigator.js

```diff
@@ -17,6 +32,11 @@ class Navigator {
 
           const { latitude, longitude, accuracy } = position.coords;
 
+          Storage.setItem(Storage.LAST_LOCATION_TIME, Date.now())
+          Storage.setItem(Storage.LATITUDE, latitude)
+          Storage.setItem(Storage.LONGITUDE, longitude)
+          Storage.setItem(Storage.ACCURACY, accuracy)
```

Comment:
I think we can just roll these all up into JSON and use one key instead of having the separate fields. Also maybe we should only cache it if the setting is enabled.
```js
if (locationTimeToLive) {
  const lastLocation = { latitude, longitude, accuracy, updatedAt: Date.now() };
  Storage.setItem(Storage.LAST_LOCATION, JSON.stringify(lastLocation));
}
```
And then when we read it out, we can do `JSON.parse`
```js
if (locationTimeToLive) {
  try {
    const lastLocation = JSON.parse(Storage.getItem(Storage.LAST_LOCATION));
    const { latitude, longitude, accuracy, updatedAt } = lastLocation;
    // check expiration stuff goes here

  } catch (e) {
    console.warn('Radar SDK: could not load cached location.');
  }
  const 
}